extension/index.js

```diff
@@ -38,10 +42,19 @@ async function init() {
     addDependencies(createBox('Dev Dependencies'), devDependencies);
   }
 
-  if (dependencies.length && !pkg.private) {
-    const link = html`<a class="npmhub-anvaka btn btn-sm">Visualize full tree`;
-    link.href = `http://npm.anvaka.com/#/view/2d/${esc(pkg.name)}`;
-    dependenciesBox.insertBefore(link, dependenciesBox.firstChild);
+  if (!pkg.private) {
+    const realPkg = await window.backgroundFetch(getPkgUrl(pkg.name));
+    if (realPkg.name) { // if 404, realPkg === {}
```

Comment:
I don't know what other results we can get, but anything else is likely to just throw an error inside .json()

Either way no button will be displayed because the error will terminate the program (and no further steps are there to be executed)

An alternative would be to drop the await and just stick to a promise there so that no uncaught errors appear in the console.